lib/screens/onboarding/show_onboarding_choice.dart

```diff
@@ -104,6 +104,15 @@ class ShowOnboardingChoice extends StatelessWidget {
     );
   }
 
+  Future<void> safeLaunch(String urlString) async {
```

Comment:
should this method be placed in a more general location? So it can be used in other places.

Reply:
Possibly... IDK, just wanted to get it working